main/src/main/java/cgeo/geocaching/utils/MapMarkerUtils.java

```diff
@@ -252,10 +253,15 @@ public static CacheMarker getWaypointMarker(final Resources res, final Waypoint
                     .append(cache.isArchived())
                     .append(cache.isLinearAlc() ? waypoint.getPrefix() : false)
                     .append(cache.getAssignedEmoji())
-                    .append(cache.getType());
+                    .append(cache.getType())
+                    .append(cache.isFound());
         }
         final int hashcode = hcb.toHashCode();
 
+        if (cache != null && StringUtils.equals(cache.getName(), "Der Hausmeister vom Palast der Republik")) {
```

Comment:
:-)

Reply:
I was mapping a part of Berlin (obviously :-)) and this virtual cache was a fine test case, its stages being shown as unfound first, but due to a found multi with stages nearby later turning into found stages :-(

Reply:
Ah. I thought it might be a comment related to bug fixing work :-)

Merry Christmas to everyone. 🎄🎁🦌⛄🎅

Reply:
> Merry Christmas to everyone. 🎄🎁🦌⛄🎅

Thanks & same to all of you @cgeo/team. Thanks for another fantastic year for c:geo, for all the dedication, thoughts and efforts you have put into the best geocaching app for Android :-)

Reply:
Thanks for all the work everyone has spent on improving cgeo during the last year... Merry Christmas! 🎅🎄🌟🎁